### ISSUE TITLE
can't share images via share dialogue

### ISSUE THREAD

**Issue by lub** [vector-im/element-android#7400]
### Steps to reproduce

1. Open Gallery
2. Select an image
3. Click the three dots
4. Select Element dbg
5. Select a room
6. The image isn't sent, instead the normal timeline view appears

#### What did you expect?

The confirmation dialogue with the "original size" check box to appear.

#### What happened instead?

It showed the normal timeline view and didn't send the image.

### Your phone model

ONEPLUS A5010

### Operating system version

Android 12

### Application version and app store

1.5.4-dev [212758362] (F-b12082) develop

It still works with this:
1.5.2 [40105020] (F-f6411d67)

### Homeserver

imninja.net

### Will you send logs?

Yes

### Are you willing to provide a PR?

No

**Comment by lephisto** [comment]
Same Issues: Oneplus 9 Pro, latest OxygenOS (Android 12)

**Comment by th0mcat** [comment]
Same, on Pixel 7 Pro and Galaxy Tab S7+

**Comment by bmarty** [comment]
Will be fixed in 1.5.6

**Comment by hho** [comment]
This is not limited to images, sharing text and URLs also doesn't work anymore.

**Comment by neonknight** [comment]
when can version 1.5.6. be expected?

**Comment by neonknight** [comment]
It seems like a minor glitch and workarounds exist, yet it is bothersome for users. I get multiple complaints per day, many people repeatedly asking when it'll be fixed

Bug has now been known for three weeks and fixed for two weeks. I see there are [new releases](https://github.com/vector-im/element-android/releases) but they don't show up in play store or f-droid. 

Therefore, let me rephrase @bmarty : when can we expect version 1.5.6 or later to hit play store?

**Comment by immanuelfodor** [comment]
What are the workarounds? It is most annoying that share to Element doesn't work

**Comment by jmartinesp** [comment]
> when can we expect version 1.5.6 or later to hit play store?

v1.5.6 hasn't been released because it has a regression that prevents syncing data with old synapse versions. There's v1.5.7, that's being slowly rolled out at the moment in Google Play if I'm not mistaken. It can also be installed from GH releases.

> What are the workarounds? It is most annoying that share to Element doesn't work

I think it's possible to launch the share screen in 1.5.6 if you rotate the phone once you're on the chat screen, but I'm not 100% sure.

**Comment by immanuelfodor** [comment]
Can I install the GH release version over the PS auto update version? Will it work with PS updates later?

**Comment by jmartinesp** [comment]
I think it was also fully released to Play Store a minute ago 😅 . It might take a while to appear to users though.

**Comment by hurradiegams** [comment]
> > when can we expect version 1.5.6 or later to hit play store?
> 
> v1.5.6 hasn't been released because it has a regression that prevents syncing data with old synapse versions. There's v1.5.7, that's being slowly rolled out at the moment in Google Play if I'm not mistaken. It can also be installed from GH releases.
> 
> > What are the workarounds? It is most annoying that share to Element doesn't work
> 
> I think it's possible to launch the share screen in 1.5.6 if you rotate the phone once you're on the chat screen, but I'm not 100% sure.

1.5.6 has been released on Monday, 7th November.
-> https://f-droid.org/en/packages/im.vector.app/

**Comment by jmartinesp** [comment]
> 1.5.6 has been released on Monday, 7th November. -> https://f-droid.org/en/packages/im.vector.app/

You're right, sorry. It hadn't been *fully released* on Play Store, but it was in F-droid. There should be an F-droid release soon.

**Comment by gitvalds** [comment]
> I think it's possible to launch the share screen in 1.5.6 if you rotate the phone once you're on the chat screen

This workaround works for me both on Element 1.5.4 on Android 7.0 (Huawei), and on Element 1.5.4 on Android 12 (Samsung).
Thanks.
